templates/base24.mustache

```diff
@@ -0,0 +1,61 @@
+# Base24 {{scheme-name}}
+# Scheme author: {{scheme-author}}
+# Template author: Tinted Theming: (https://github.com/tinted-theming)
+
+# default statusbar colors
+set-option -g status-style "fg=#{{base04-hex}},bg=#{{base01-hex}}"
+
+# default window title colors
+set-window-option -g window-status-style "fg=#{{base04-hex}},bg=#{{base01-hex}}"
+
+# active window title colors
+set-window-option -g window-status-current-style "fg=#{{base13-hex}},bg=#{{base01-hex}}"
+
+# pane border
+set-option -g pane-border-style "fg=#{{base01-hex}}"
+set-option -g pane-active-border-style "fg=#{{base05-hex}}"
```

Comment:
curious on the reason for updating base04 to base05 here? :)

Reply:
I'm not sure actually, I think it was a mistake, I'll revert.